src/device/gcn_intrinsics/math.jl

```diff
@@ -0,0 +1,203 @@
+# @inline round(x::Float32) = @wrap __nv_llroundf(x::float)::i64
+# @inline round(x::Float64) = @wrap __nv_round(x::double)::double
+# @inline round(x::Float32) = @wrap __nv_roundf(x::float)::float
+
+# TODO: differentiate in return type, map correctly
+# @inline rint(x::Float64) = @wrap __nv_llrint(x::double)::i64
+# @inline rint(x::Float32) = @wrap __nv_llrintf(x::float)::i64
+# @inline rint(x::Float64) = @wrap __nv_rint(x::double)::double
+# @inline rint(x::Float32) = @wrap __nv_rintf(x::float)::float
+
+# TODO: would conflict with trunc usage in this module
+# @inline trunc(x::Float64) = @wrap __nv_trunc(x::double)::double
+# @inline trunc(x::Float32) = @wrap __nv_truncf(x::float)::float
+
+#= TODO: Wrap me!
+@inline min(x::Int32, y::Int32) = @wrap __nv_min(x::i32, y::i32)::i32
+@inline min(x::Int64, y::Int64) = @wrap __nv_llmin(x::i64, y::i64)::i64
+@inline min(x::UInt32, y::UInt32) = convert(UInt32, @wrap __nv_umin(x::i32, y::i32)::i32)
+@inline min(x::UInt64, y::UInt64) = convert(UInt64, @wrap __nv_ullmin(x::i64, y::i64)::i64)
+@inline min(x::Float64, y::Float64) = @wrap __nv_fmin(x::double, y::double)::double
+@inline min(x::Float32, y::Float32) = @wrap __nv_fminf(x::float, y::float)::float
+
+@inline max(x::Int32, y::Int32) = @wrap __nv_max(x::i32, y::i32)::i32
+@inline max(x::Int64, y::Int64) = @wrap __nv_llmax(x::i64, y::i64)::i64
+@inline max(x::UInt32, y::UInt32) = convert(UInt32, @wrap __nv_umax(x::i32, y::i32)::i32)
+@inline max(x::UInt64, y::UInt64) = convert(UInt64, @wrap __nv_ullmax(x::i64, y::i64)::i64)
+@inline max(x::Float64, y::Float64) = @wrap __nv_fmax(x::double, y::double)::double
+@inline max(x::Float32, y::Float32) = @wrap __nv_fmaxf(x::float, y::float)::float
+
+@inline saturate(x::Float32) = @wrap __nv_saturatef(x::float)::float
+
+
+## division and remainder
+
+@inline mod(x::Float64, y::Float64) = @wrap __nv_fmod(x::double, y::double)::double
+@inline mod(x::Float32, y::Float32) = @wrap __nv_fmodf(x::float, y::float)::float
+
+@inline rem(x::Float64, y::Float64) = @wrap __nv_remainder(x::double, y::double)::double
+@inline rem(x::Float32, y::Float32) = @wrap __nv_remainderf(x::float, y::float)::float
+
+@inline div_fast(x::Float32, y::Float32) = @wrap __nv_fast_fdividef(x::float, y::float)::float
+
+## Bessel
+
+@inline jn(n::Int32, x::Float64) = @wrap __nv_jn(n::i32, x::double)::double
+@inline jn(n::Int32, x::Float32) = @wrap __nv_jnf(n::i32, x::float)::float
+
+@inline y0(x::Float64) = @wrap __nv_y0(x::double)::double
+@inline y0(x::Float32) = @wrap __nv_y0f(x::float)::float
+
+@inline y1(x::Float64) = @wrap __nv_y1(x::double)::double
+@inline y1(x::Float32) = @wrap __nv_y1f(x::float)::float
+
+
+## distributions
+
+@inline normcdf(x::Float64) = @wrap __nv_normcdf(x::double)::double
+@inline normcdf(x::Float32) = @wrap __nv_normcdff(x::float)::float
+
+@inline normcdfinv(x::Float64) = @wrap __nv_normcdfinv(x::double)::double
+@inline normcdfinv(x::Float32) = @wrap __nv_normcdfinvf(x::float)::float
```

Comment:
Let's remove these commented out pieces

Reply:
I was just keeping them around as a reminder of what intrinsics we're missing to be up-to-date with CUDAnative, since there's a lot of them.